### PR TITLE
Pzuraq reactivity blog posts

### PR DESCRIPTION
**Pull request by jaredgalanis**
adds the first two blog posts.

### CODE REVIEW

File: source/2020-02-14-the-ember-times-issue-135.md [hunk 1]
```diff
@@ -25,13 +26,21 @@ To check out how to upgrade your app to 3.15, check out this [great Discord thre
 
 ---
 
-## [Section title in sentence case 🐹](#section-url)
+## [Autotracking! A Blog Series on Reactivity Systems 🤖](https://www.pzuraq.com/thinking-with-autotracking-what-makes-a-good-reactive-system/)
 
-<change section title emoji>
-<consider adding some bold to your paragraph>
+Interested in learning more about auto tracking, Ember Octane's new reactivity system? Well then, you're in luck 🍀. [Chris Garrett @pzuraq](https://github.com/pzuraq) is starting another super interesting new blog post series on that exact subject!
```
Comment (ijlee2) [inline source/2020-02-14-the-ember-times-issue-135.md]: I removed the space for consistency.
```suggestion
Interested in learning more about autotracking, Ember Octane's new reactivity system? Well then, you're in luck 🍀. [Chris Garrett (@pzuraq)](https://github.com/pzuraq) is starting another super interesting new blog post series on that exact subject!
```

File: source/2020-02-14-the-ember-times-issue-135.md [hunk 2]
```diff
@@ -25,13 +26,21 @@ To check out how to upgrade your app to 3.15, check out this [great Discord thre
...
 
-<add your name to author list, top and bottom>
-<add blurb and emoji to "SOME-INTRO-HERE">
+The [first post](https://www.pzuraq.com/thinking-with-autotracking-what-makes-a-good-reactive-system/) came out at the end of last month and it examines what reactivity means, how it relates to declarative and imperative programming models. Even experienced Ember developers will likely get a lot of value out of reading about what these often-used, but rarely fully understood, terms mean.
```
Comment (ijlee2) [inline source/2020-02-14-the-ember-times-issue-135.md]:
```suggestion
The [first post](https://www.pzuraq.com/thinking-with-autotracking-what-makes-a-good-reactive-system/), which came out at the end of last month, examines what reactivity means and how it relates to declarative and imperative programming models. Even experienced Ember developers will likely get a lot of value out of reading about what these often-used, but rarely fully understood, terms mean. 
```

File: source/2020-02-14-the-ember-times-issue-135.md [hunk 3]
```diff
@@ -25,13 +26,21 @@ To check out how to upgrade your app to 3.15, check out this [great Discord thre
...
 
-<change section title emoji>
-<consider adding some bold to your paragraph>
+Interested in learning more about auto tracking, Ember Octane's new reactivity system? Well then, you're in luck 🍀. [Chris Garrett @pzuraq](https://github.com/pzuraq) is starting another super interesting new blog post series on that exact subject!
 
-<add your name to author list, top and bottom>
-<add blurb and emoji to "SOME-INTRO-HERE">
+The [first post](https://www.pzuraq.com/thinking-with-autotracking-what-makes-a-good-reactive-system/) came out at the end of last month and it examines what reactivity means, how it relates to declarative and imperative programming models. Even experienced Ember developers will likely get a lot of value out of reading about what these often-used, but rarely fully understood, terms mean. 
+
+Explaining things in an approachable and conversational style, [@pzuraq](https://github.com/pzuraq) also looks at how state is used in these programming models and how it is used in functional programming models. It’s a great intro for any one looking to learn a bit about reactivity systems in general.
```
Comment (ijlee2) [inline source/2020-02-14-the-ember-times-issue-135.md]:
```suggestion
In an approachable and conversational style, [@pzuraq](https://github.com/pzuraq) also explains how state is used in these programming models as well as functional programming models. It’s a great intro for anyone looking to learn a bit about reactivity systems.
```

File: source/2020-02-14-the-ember-times-issue-135.md [hunk 4]
```diff
@@ -25,13 +26,21 @@ To check out how to upgrade your app to 3.15, check out this [great Discord thre
...
+
+Bonus 🎉 [@pzuraq’s](https://github.com/pzuraq) [second post](https://www.pzuraq.com/thinking-with-autotracking-what-makes-a-good-reactive-system/) about what makes a good reactivity system came out this week!
+
+In this second post [@pzuraq](https://github.com/pzuraq) dives deeper and surveys several reactivity systems. Compaing HTML/CSS (did you know they are reactive!?), push-based reactivity systems that are often implemented with JavaScript (such as Ember Classic and other systems like the Observable model), and pull-based reactivity systems of React and the Virtual DOM, Vue and Elm.
```
Comment (ijlee2) [inline source/2020-02-14-the-ember-times-issue-135.md]: I found the second sentence a bit hard to parse. If I understood the sentence right, I think making a list like this may help:
```suggestion
[@pzuraq](https://github.com/pzuraq) dives deeper and surveys several reactivity systems:
- HTML/CSS (did you know they are reactive!?)
- Push-based reactivity systems, which are often implemented with JavaScript (e.g. Ember Classic and other systems like the Observable model)
- Pull-based reactivity systems (e.g. React and the Virtual DOM, Vue and Elm)
```

File: source/2020-02-14-the-ember-times-issue-135.md [hunk 5]
```diff
@@ -25,13 +26,21 @@ To check out how to upgrade your app to 3.15, check out this [great Discord thre
...
+
+Explaining things in an approachable and conversational style, [@pzuraq](https://github.com/pzuraq) also looks at how state is used in these programming models and how it is used in functional programming models. It’s a great intro for any one looking to learn a bit about reactivity systems in general.
+
+Bonus 🎉 [@pzuraq’s](https://github.com/pzuraq) [second post](https://www.pzuraq.com/thinking-with-autotracking-what-makes-a-good-reactive-system/) about what makes a good reactivity system came out this week!
+
+In this second post [@pzuraq](https://github.com/pzuraq) dives deeper and surveys several reactivity systems. Compaing HTML/CSS (did you know they are reactive!?), push-based reactivity systems that are often implemented with JavaScript (such as Ember Classic and other systems like the Observable model), and pull-based reactivity systems of React and the Virtual DOM, Vue and Elm. 
+
+There's some helpful diagrams and clear explanations to help you better understand the finer points. If you’re looking to learn more about how these models of reactivity are similar and different, then this post is a great read for you.
```
Comment (ijlee2) [inline source/2020-02-14-the-ember-times-issue-135.md]:
```suggestion
There are helpful diagrams and clear explanations to help you better understand the finer points. If you’re looking to learn more about how these models of reactivity are similar and different, then this post is a great read for you.
```

File: source/2020-02-14-the-ember-times-issue-135.md [hunk 6]
```diff
@@ -25,13 +26,21 @@ To check out how to upgrade your app to 3.15, check out this [great Discord thre
...
+
+Be sure to be on the look out for more updates from the Ember Times team as more of these awesome blog posts are released!
```
Comment (ijlee2) [inline source/2020-02-14-the-ember-times-issue-135.md]:
```suggestion
Be sure to be on the lookout for more updates from the Ember Times team as more of these awesome blog posts are released!
```

**Review by ijlee2 — APPROVED** [review]
Thanks for reading and covering both articles! I left a few suggestions that I thought would help a reader understand your points.

**Comment by jaredgalanis** [comment]
> Thanks for reading and covering both articles! I left a few suggestions that I thought would help a reader understand your points.

Thank you @ijlee2 ! All looked good to me so I committed the suggestions.

**Review by MinThaMie — APPROVED** [review]
